crates/holochain/tests/tests/publish/mod.rs

```diff
@@ -8,7 +7,10 @@ use holochain_zome_types::validate::ValidationReceiptSet;
 /// Verifies that publishing terminates naturally when enough validation receipts are received.
 #[cfg(feature = "test_utils")]
 #[tokio::test(flavor = "multi_thread")]
-#[ignore = "receipt completion is flaky, which has been rechecked since the workflow reviews"]
+#[cfg_attr(
+    not(target_os = "linux"),
+    ignore = "seems to actually work on macos wasmer_sys as well, but we don't have a great way to exclude only wasmer_wamr in CI"
```

Comment:
Doesn't it have a feature flag? I think it does on `holochain`

Reply:
Oh, cool. Done: https://github.com/holochain/holochain/pull/4834/commits/463b1f93228ce3c9aed7f06bbd4251d71f941df7